src/worker_loader.js

```diff
@@ -28,7 +28,6 @@ importScripts('./shared/compatibility.js');
 importScripts('../node_modules/systemjs/dist/system.js');
 importScripts('../systemjs.config.js');
 
-Promise.all([SystemJS.import('pdfjs/core/network'),
-             SystemJS.import('pdfjs/core/worker')]).then(function () {
+Promise.resolve(SystemJS.import('pdfjs/core/worker')).then(function () {
```

Comment:
Here and in api.js, you can directly use `SystemJS.import` without wrapping with `Promise.resolve`:
```
SystemJS.import('pdfjs/core/worker').then(function () {...
```